### PR TITLE
[VEN-2671]: VTreasuryV8 deployment on zksyncsepolia

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves #

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: hardhat.config.zksync.ts [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            enabled: true,
+            runs: 200,
+          },
+          outputSelection: {
+            "*": {
+              "*": ["storageLayout"],
+            },
+          },
+        },
+      },
```
Comment (chechu) [inline hardhat.config.zksync.ts:61]: Are we deploying solidity 0.5.16 contracts to zkSync?

File: hardhat.config.zksync.ts [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+};
+
+function isFork() {
+  return process.env.FORK === "true"
+    ? {
+        allowUnlimitedContractSize: false,
+        loggingEnabled: false,
+        forking: {
+          url: process.env[`ARCHIVE_NODE_${process.env.FORKED_NETWORK}`] || "https://sepolia.era.zksync.dev",
+          blockNumber: 21068448, // TODO: change this
```
Comment (Debugger022) [inline hardhat.config.zksync.ts:141]: Is there any change to be done here?
```suggestion
          blockNumber: 21068448,
```

File: hardhat.config.zksync.ts [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+          outputSelection: {
+            "*": {
+              "*": ["storageLayout"],
+            },
+          },
+        },
+      },
+    ],
+  },
+  zksolc: {
```
Comment (Debugger022) [inline hardhat.config.zksync.ts:79]: zksolc object is already defined above, is there any logic behind defining it again.

File: hardhat.config.zksync.ts [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    deployments: {},
+  },
+  docgen: {
+    outputDir: "./docgen-docs",
+    pages: "files",
+    templates: "docgen-templates",
+  },
+};
+
+function isFork() {
```
Comment (Debugger022) [inline hardhat.config.zksync.ts:134]: Do we need this function here too? As it is already present in `hardhat.config.ts`.

File: hardhat.config.zksync.ts (hunk 1 above)
Comment (GitGuru7) [reply hardhat.config.zksync.ts:61]: I don't think so, will remove this.

File: hardhat.config.zksync.ts (hunk 4 above)
Comment (GitGuru7) [reply hardhat.config.zksync.ts:134]: Yes, we need this for zksync, whenever we want to run fork tests on zksync.

File: hardhat.config.zksync.ts (hunk 2 above)
Comment (GitGuru7) [reply hardhat.config.zksync.ts]: Yes, block number needs to be updated.
Done [a4061b3](https://github.com/VenusProtocol/venus-protocol/pull/500/commits/a4061b3c612ea137697707f396388ad7551e6ff6)

**Review by chechu — APPROVED** [review]

**Pull request merged**